nilearn/maskers/_utils.py

```diff
@@ -125,3 +125,32 @@ def concatenate_surface_images(imgs):
     output = SurfaceImage(mesh=imgs[0].mesh, data=output_data)
 
     return output
+
+
+def check_surface_data_ndims(img, dim, var_name="img"):
```

Comment:
Added this function to validate the dimensions of input images, and I noticed we are not doing that in SurfaceLabelsMasker.